src/map/handler/TouchZoom.js

```diff
@@ -11,10 +11,18 @@ L.Map.mergeOptions({
     // Whether the map can be zoomed by touch-dragging with two fingers. If
     // passed `'center'`, it will zoom to the center of the view regardless of
     // where the touch events (fingers) were. Enabled for touch-capable web
-    // browsers except for old Androids.
-    touchZoom: L.Browser.touch && !L.Browser.android23,
+    // browsers.
+    touchZoom: Browser.touch,
 
+    /**
+     * @TODO check if this is a duplicate of `L.Map.TouchGestures::bounceAtZoomLimits`
+     */
+
+    // @option bounceAtZoomLimits: Boolean = true
+    // Set it to false if you don't want the map to zoom beyond min/max zoom
+    // and then bounce back when pinch-zooming.
     bounceAtZoomLimits: false,
```

Comment:
@heyajohnny looks so much like a duplicate of: https://github.com/Raruto/leaflet-rotate/blob/6bfcccecf50143fd927e5f347a3f4d5a930c2f56/src/map/handler/TouchGestures.js#L12

BTW, could be related to https://github.com/Raruto/leaflet-rotate/issues/19? (eg. due to current incorrect map bounds calculations, ref: https://github.com/Raruto/leaflet-rotate/issues/18#issuecomment-1399349255)

Reply:
Yes it's a duplicate of 'TouchGestures.js' and this one (in 'TouchZoom.js') can be removed.
I've tested it with these combinations:
1. touchGestures: true, touchZoom: false
2. touchGestures: false, touchZoom: true
3. touchGestures: true, touchZoom: true

And it works as expected without this 'bounceAtZoomLimits: false,'.